Neos.Neos/Classes/Domain/Model/WorkspacePermissions.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Neos\Neos\Domain\Model;
+
+use Neos\Flow\Annotations as Flow;
+
+/**
+ * @api
+ */
+#[Flow\Proxy(false)]
+final readonly class WorkspacePermissions
+{
+    private function __construct(
+        public bool $read,
+        public bool $write,
+        public bool $manage,
```

Comment:
what is the meaning of `false, false, true` (if the user is admin he can manage ... but what is manage again? ^^) i think we need here a doc comment as well *g